### PR TITLE
[DOCFIX] Add footprint to UFS docs

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/en/ufs/GlusterFS.md [hunk 1]
```diff
@@ -66,3 +66,9 @@ To stop Alluxio, you can run:
 ```console
 $ ./bin/alluxio-stop.sh local
 ```
+
+## Contributed by Alluxio Community
+
+GlusterFS UFS integration is contributed and maintained by Alluxio community.
+Source code locates at [here](https://github.com/Alluxio/alluxio-extensions/tree/master/underfs/glusterfs).
```
Comment (Xenorith) [inline docs/en/ufs/GlusterFS.md:73]: `The source code is located [here](link).`

File: docs/en/ufs/GlusterFS.md [hunk 2]
```diff
@@ -66,3 +66,9 @@ To stop Alluxio, you can run:
 ```console
 $ ./bin/alluxio-stop.sh local
 ```
+
+## Contributed by Alluxio Community
+
+GlusterFS UFS integration is contributed and maintained by Alluxio community.
```
Comment (Xenorith) [inline docs/en/ufs/GlusterFS.md:72]: `by the Alluxio community.`

File: docs/en/ufs/GlusterFS.md [hunk 3]
```diff
@@ -66,3 +66,9 @@ To stop Alluxio, you can run:
...
+Source code locates at [here](https://github.com/Alluxio/alluxio-extensions/tree/master/underfs/glusterfs).
+Welcome to submit pull requests to improve the integration!
```
Comment (Xenorith) [inline docs/en/ufs/GlusterFS.md:74]: `Feel free submit pull requests to improve the integration and update the documentation [here](https://github.com/Alluxio/alluxio/edit/master/docs/en/ufs/GlusterFS.md) if any information is missing or out of date.`

**Review by Xenorith — CHANGES_REQUESTED** [review]
i commented on one doc, but applies to all docs

**Review by Xenorith — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**